docs/en/integrations/index.mdx

```diff
@@ -167,7 +168,9 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |Supabase|<img src={require('./images/logos/logo_supabase.png').default} class="image" alt="Supabase logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion|Open source Firebase alternative|[GitHub](https://github.com/supabase/wrappers/tree/main/wrappers/src/fdw/clickhouse_fdw),[Blog](https://clickhouse.com/blog/migrating-data-between-clickhouse-postgres)|
 |TABLUM.IO|<TablumSVG style={{width: '3rem'}} />|SQL client|TABLUM.IO ingests data from a variety of sources, normalizes and cleans inconsistencies,  and gives you access to it via SQL.|[Documentation](/docs/en/integrations/sql-clients/tablum.md)|
 |Tooljet|<img src={require('./images/logos/tooljet.png').default} class="image" alt="Tooljet logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data Visualization|ToolJet is an open-source low-code framework to build and deploy custom internal tools.|[Documentation](https://docs.tooljet.com/docs/1.x.x/data-sources/clickhouse/)|
+|Upstash|<img src={require('./images/logos/upstash.svg').default} class="image" alt="Upstash logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data Ingestion|A data platform offering serverless Kafka and other solutions|[Website](https://upstash.com/)|
 |Vector|<img src={require('./images/logos/vector.png').default} class="image" alt="Vector logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data ingestion|A lightweight, ultra-fast tool for building observability pipelines with built-in compatibility with ClickHouse.|[Documentation](/docs/en/integrations/vector/)|
+|WarpStream|<Image>|Data Ingestion|A Kafka compatible data streaming platform built directly on top of object storage|[Website](https://www.warpstream.com/)|
```

Comment:
<Image> is a temporary placeholder, isn't it?

Reply:
Thanks, guys! I staged this PR while I rushed off to a meeting. The doc updates look good now.